### DISCUSSION TITLE
Disable HX-Boost per request

### DISCUSSION THREAD

**Discussion by frenetisch-applaudierend** [bigskysoftware/htmx#2392] [Q&A] [answered]
In my application I have added `hx-boost="true"` to the body of my HTML. Now everytime I do some request, it gets boosted. So far this is intended. However also when I do a redirect from the server, this redirect gets boosted. Mostly that is not a problem, but now I have a redirect to a third party site which is not allowing my site in their CORS policy. Therefore the browser rejects the AJAX request. When I disable hx-boost the redirect works.

Hence the question, is there a HX-* header or similar that I can set with the response, to disable hx-boost for this specific response? Or is there another workaround that enables this CORS request?

**Comment by andryyy** [comment, marked as answer]
Is adding hx-boost="false" to the element an acceptable solution?

**Reply by andryyy** [reply]
A 30x as redirect should also go undercover of HTMX (your browser acts on it).

**Reply by frenetisch-applaudierend** [reply]
Thank you! Indeed hx-boost on the initial link solves it and is more than acceptable
